### PR TITLE
A new feature for the b2b-samples/guardio-insurance-sample-application

### PR DESCRIPTION
**Pull request by Achintha444**

### CODE REVIEW

**Review by thivi — APPROVED** [review]

File: b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    } else {
+        if (session.expires || session.error) {
+            return redirect('/500');
+        } else {
+            return redirect('/404');
+        }
+    }
+}
+
+export default function Org(props) {
```
Comment (sandushi) [inline b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js:51]: Let's start function name in a lowrercase letter

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/dashboard/decodeMe.js [hunk 2]
```diff
@@ -21,7 +21,12 @@ import decodeUser from "../../util/apiUtil/decodeUser";
 
 export default async function decodeMe(session) {
     try {
-        const meData = await callMe(session);
+        var meData;
+        if(session.user.id==undefined || session.user.userName==undefined || session.user.name==undefined
```
Comment (sandushi) [inline b2b-sample/guardio-insurance-sample-application/util/apiDecode/dashboard/decodeMe.js:25]: formatting issue . space before and after `==`. Check other places as well.

File: b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+                alignItems: 'center'
+            }
+        }>
+            <p style={
+                {
+                    fontSize: '2em'
+                }
+            }>You will be redirected to the {props.orgName} login page</p>
+        </div>
+
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js:84]: Shall we remove the unwanted new lines? Check other places too.

File: b2b-sample/guardio-insurance-sample-application/util/util/orgUtil/orgUtil.js [hunk 4]
```diff
@@ -57,6 +57,16 @@ function getOrgIdfromRouterQuery(routerQuery) {
     return undefined;
 }
 
+function getOrgIdFromQuery(query) {
```
Comment (lashinijay) [inline b2b-sample/guardio-insurance-sample-application/util/util/orgUtil/orgUtil.js:60]: Can we simplify this method?

Suggestion
```
function getOrgIdFromQuery(query) {
    const org = config.SAMPLE_ORGS.find(org => org.routerQuery === query);
    return org ? org.id : null;
}
```

File: b2b-sample/guardio-insurance-sample-application/util/apiDecode/dashboard/decodeMe.js (hunk 2 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/apiDecode/dashboard/decodeMe.js:25]: Fixed with https://github.com/Achintha444/samples-is/commit/c56439215193067034ac953d02cb38272b2be8d4#diff-41837f5472f12624dbae0a8457f15e50adb73bf6a7f5f2aac6228a4616a1d56f

File: b2b-sample/guardio-insurance-sample-application/util/util/orgUtil/orgUtil.js (hunk 4 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/util/util/orgUtil/orgUtil.js:60]: Fixed with
https://github.com/Achintha444/samples-is/commit/869e218ddd640521a7a5629b4ba6bf12e1e1afea#diff-54e1f2d15b1f45bc469f601fb94904610dd9f62dea8bcc1a3f2a64a73686ff29

File: b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js (hunk 3 above)
Comment (Achintha444) [reply b2b-sample/guardio-insurance-sample-application/pages/o/moveOrg.js:84]: Fixed with https://github.com/Achintha444/samples-is/commit/571af28362d9fee354f56a14fb31f1903b666112#diff-d6d4c1d46f50b963c040d2902bb637934a843ced966656fd231830ea89453967

**Pull request merged**